### PR TITLE
GCP/ASM - EXPANDR-3281

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/CIAC-5981
https://jira-hq.paloaltonetworks.local/browse/EXPANDR-3281

## Description
This is effectively updating playbooks after integration change in https://jira-hq.paloaltonetworks.local/browse/CIAC-5981

## Screenshots
video

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [X] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@dansterenson , shared demo video with you via google drive

**Review by dansterenson — CHANGES_REQUESTED** [review]
Nice, see my notes. @efelmandar still needs to review the playbooks.

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_1.md
[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_1.md:1]: @dansterenson , i fixed a typo in this release note.  I could create a separate PR, but it doesn't make sense for me to.

File: Packs/GCP-Enrichment-Remediation/ReleaseNotes/1_1_1.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+##### GCP - Enrichment
+- Updated the playbook to incorporate changes from https://github.com/demisto/content/pull/25410.
+##### GCP - Firewall Remediation
+- Updated the playbook to incorporate changes from https://github.com/demisto/content/pull/25410.
```
[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/GCP-Enrichment-Remediation/ReleaseNotes/1_1_1.md]: [update GCP RN and redo Remediation play](https://github.com/demisto/content/pull/25484/commits/4efa8682fb788103f06c2c4962b1a481f2450b2b)

File: Packs/GCP-Enrichment-Remediation/ReleaseNotes/1_1_1.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+##### GCP - Enrichment
+- Updated the playbook to incorporate changes from https://github.com/demisto/content/pull/25410.
```
[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/GCP-Enrichment-Remediation/ReleaseNotes/1_1_1.md]: [update GCP RN and redo Remediation play](https://github.com/demisto/content/pull/25484/commits/4efa8682fb788103f06c2c4962b1a481f2450b2b)

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_3.md
[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_3.md:1]: [update GCP RN and redo Remediation play](https://github.com/demisto/content/pull/25484/commits/4efa8682fb788103f06c2c4962b1a481f2450b2b)

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_3.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Cortex ASM - Remediation
+
+- Added the value to the new 'GcpProject' input required by the 'GCP - Firewall Remediation' playbook.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_3.md:6]:
```suggestion
Added the value to the new 'GcpProject' input required by the **GCP - Firewall Remediation** playbook.
```

File: Packs/GCP-Enrichment-Remediation/Playbooks/GCP_-_Firewall_Remediation.yml [hunk 4]
```diff
@@ -735,7 +756,19 @@ inputs:
...
   playbookInputQuery:
+- key: GcpProject
+  value:
+    complex:
+      root: alert.asmcloud
+      accessor: Project
+      transformers:
+      - operator: FirstArrayElement
+  required: true
+  description: The name of the GCP project associated the instance and related objects.
```
Comment (ShirleyDenkberg) [inline Packs/GCP-Enrichment-Remediation/Playbooks/GCP_-_Firewall_Remediation.yml:767]:
```suggestion
  description: The name of the GCP project associated with the instance and related objects.
```

File: Packs/GCP-Enrichment-Remediation/Playbooks/GCP_-_Firewall_Remediation_README.md [hunk 5]
```diff
@@ -28,11 +34,15 @@ This playbook does not use any scripts.
 | GcpNetwork | The VPC network of the GCP instance. |  | Required |
 | RemotePort | The remote port that is publicly exposed to. | alert.remoteport | Required |
 | RemoteProtocol | The remote protocol that is publicly exposed to. |  | Required |
+| GcpProject | The name of the GCP project associated the instance and related objects. | alert.asmcloud.Project | Required |
```
Comment (ShirleyDenkberg) [inline Packs/GCP-Enrichment-Remediation/Playbooks/GCP_-_Firewall_Remediation_README.md:37]:
```suggestion
| GcpProject | The name of the GCP project associated with the instance and related objects. | alert.asmcloud.Project | Required |
```

File: Packs/GCP-Enrichment-Remediation/ReleaseNotes/1_1_1.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+##### GCP - Enrichment
+- Updated the playbook to incorporate *Google Cloud Compute* integration improvements.
```
Comment (ShirleyDenkberg) [inline Packs/GCP-Enrichment-Remediation/ReleaseNotes/1_1_1.md:4]:
```suggestion
Updated the playbook to incorporate the *Google Cloud Compute* integration improvements.
```

File: Packs/GCP-Enrichment-Remediation/ReleaseNotes/1_1_1.md [hunk 7]
```diff
@@ -0,0 +1,6 @@
...
+##### GCP - Firewall Remediation
+- Updated the playbook to incorporate *Google Cloud Compute* integration improvements.
```
Comment (ShirleyDenkberg) [inline Packs/GCP-Enrichment-Remediation/ReleaseNotes/1_1_1.md:6]:
```suggestion
Updated the playbook to incorporate the *Google Cloud Compute* integration improvements.
```

**Comment by ShirleyDenkberg** [comment]
@dansterenson @efelmandar Doc review completed.

**Review by dansterenson — APPROVED** [review]

**Pull request merged**